qemu/tests/verify_panic_status_with_pvpanic.py

```diff
@@ -19,16 +24,41 @@ def run(test, params, env):
     trigger_crash = params["trigger_crash"]
     qmp_check_info = params["qmp_check_info"]
     check_info = params.get("check_info")
+    is_aarch64 = params.get("vm_arch_name") == "aarch64"
+    check_pci_cmd = params.get("check_pci_cmd")
+    check_capability_cmd = params.get("check_capability_cmd")
+
+    # trigger kernel panic config
+    trigger_kernel_panic = params.get("trigger_kernel_panic")
+    username = params.get('username')
+    password = params.get('password')
+    port = params.get('file_transfer_port')
+    guest_path = params.get('guest_path')
+    cmd_yum_deps = params.get('cmd_yum_deps')
+    cmd_make = params.get('cmd_make')
+    io_timeout = params.get_numeric('io_timeout')
+
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
+    host_addr = vm.get_address()
```

Comment:
Should be `guest_addr/ip`?

Reply:
oh yes, thanks for reminding me. update later

Reply:
Corrected